### PR TITLE
[#1375] Add tests for StepIndicator component

### LINKED ISSUE #1375: UNIT TESTS - components/StepIndicator
## What

Create unit tests using [Jest](https://jestjs.io/en/) and [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/).

Here are a couple examples of how unit tests should be created:

1. [Unit tests for our A component](https://github.com/thenewboston-developers/Website/blob/development/src/components/A/A.test.tsx)
2. [Unit tests for Icon component](https://github.com/thenewboston-developers/ui-js/blob/master/src/components/Icon/Icon.test.tsx)
3. [Unit tests for Button component](https://github.com/thenewboston-developers/ui-js/blob/master/src/components/Inputs/Button/Button.test.tsx)

## General Acceptance Criteria for writing Unit tests

1. Should create a `[nameOfComponent].test.tsx` within the same directory of the component.
2. Should have tests that tests every `prop` of the component.
3. If there are any conditional logic within the component, there should be unit tests that accounts for each conditional branches.
4. `npm run test` should be passing.

## Specific Acceptance Criteria

1. make sure you tests for each prop

### PR DESCRIPTION
**Pull request by jamessspanggg**
Fixes #1375 

Acc No: c54c04774efaae20746fd3f29cdd619fea512e119bc1082b863572b2e8844104

### CODE REVIEW

**Review by thesanjeevsharma — APPROVED** [review]

File: src/components/StepIndicator/StepIndicator.test.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    const el = screen.getByTestId('StepIndicator');
+
+    expect(el.className).toBe('StepIndicator');
+  });
+
+  it('renders with classNames passed in', () => {
+    render(<StepIndicator className="test" {...baseProps} />);
+    const el = screen.getByTestId('StepIndicator');
+
+    expect(el.className).toContain('test');
```
Comment (angle943) [inline src/components/StepIndicator/StepIndicator.test.tsx:24]: let's add tests here to make sure all the children div has correct classNames too

File: src/components/StepIndicator/StepIndicator.test.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  number: 1,
+  text: <div>Hello world</div>,
+};
+
+describe('StepIndicator', () => {
+  it('renders proper default className', () => {
+    render(<StepIndicator {...baseProps} />);
+    const el = screen.getByTestId('StepIndicator');
+
+    expect(el.className).toBe('StepIndicator');
```
Comment (angle943) [inline src/components/StepIndicator/StepIndicator.test.tsx:17]: can you add tests here for the children div's too?

Comment (jamessspanggg) [reply src/components/StepIndicator/StepIndicator.test.tsx:19]: yep done so

File: src/components/StepIndicator/StepIndicator.test.tsx (hunk 1 above)
Comment (jamessspanggg) [reply src/components/StepIndicator/StepIndicator.test.tsx:30]: done

**Comment by angle943** [comment]
@jamessspanggg updated the test to test for the `bemify` classNames

**Review by angle943 — APPROVED** [review]

**Pull request merged**